csds-client/main.go

```diff
@@ -2,11 +2,55 @@ package main
 
 import (
 	"envoy-tools/csds-client/client"
+	"flag"
 	"log"
+	"time"
 )
 
+// ParseFlags parses flags to ClientOptions
+func ParseFlags() client.ClientOptions {
```

Comment:
How about GetClientOptionsFromFlags? This way function name is self-documenting and when I see it being called from code - I immediately understand what it does.

Reply:
Renamed.